experiments/commands.py

```diff
@@ -0,0 +1,234 @@
+r"""Generate commands to run training scripts with different configs.
+
+This program takes as input a set of config files and outputs a command
+for each config file. Each command runs a training script using the
+associated config file. By default, each command uses the same random seed
+for training. However, if multiple random seeds are specified, the program
+generates multiple commands for each config file (one for each random seed).
+The commands can be executed by copying and pasting them into a command line
+or by piping them to another command line utility. This program is helpful
+for running a large number of training scripts with different config files
+using multiple random seeds.
+
+For example, we can run:
+
+python commands.py \
+    --name=run0 \
+    --cfg_pattern=../benchmarking/*ai*_seals_walker_*.json \
```

Comment:
To match `gail` and `airl`? Cute, but if `(gail|airl)` works this would be more explicit intent.

Reply:
That doesn't work. Globbing has limited expressiveness. I could use regular expressions instead, but that seems like overkill since you can usually create a directory with the config files that you want. Thoughts?

Reply:
I see, let's leave it just using globbing for now then, someone can always run `commands.py` multiple times to express things like this if needed.